cmd/minikube/cmd/start.go

```diff
@@ -1560,7 +1560,7 @@ func validateRegistryMirror() {
 // args match the format of registry.cn-hangzhou.aliyuncs.com/google_containers
 // also "<hostname>[:<port>]"
 func validateImageRepository(imageRepo string) (validImageRepo string) {
-	expression := regexp.MustCompile(`^(?:(\w+)\:\/\/)?([-a-zA-Z0-9]{1,}(?:\.[-a-zA-Z]{1,}){0,})(?:\:(\d+))?(\/.*)?$`)
+	expression := regexp.MustCompile(`^(?:(\w+)\:\/\/)?([-a-zA-Z0-9]{1,}(?:\.[-a-zA-Z0-9]{1,}){0,})(?:\:(\d+))?(\/.*)?$`)
```

Comment:
@justinmchase thank you for this PR, how about we add a unit test for this function and add all the valid cases including the one you mentioned in the PR description

Reply:
Fortunately there already was a test with a list of test cases, I was able to easily just add the new domain to the list and it succeeded locally.

Reply:
Actually, it always match, also see https://github.com/kubernetes/minikube/issues/14698#issuecomment-1426707622

Reply:
In fact, I don't quite understand the role of this validateImageRepository. If the imageRepository I configure is wrong, then I just need it to fail to pull the image and know that I configured it correctly.

So + 1 from me.